docs.html

```diff
@@ -49,7 +49,19 @@
 <h2>
 <a name="migrating" class="anchor" href="#migrating"><span class="octicon octicon-link"></span></a>Migrating a module to puppet-community</h2>
 
-<p> TODO: add docs on how to migrate a module </p>
+<p>The general flow is to…</p>
+
+<ul>
+  <li>Ask one of the Administrators to add you as a collaborator.</li>
```

Comment:
hehe.. kk ill do it. We do still need a line having the original module author add the coloaborators team to write access to the module, I think.